doc/services/pm/device.rst

```diff
@@ -56,20 +92,30 @@ suspended.
    As functions in this context cannot block, transitions that intend to use blocking
    APIs **must** check whether they can do so with :c:func:`k_can_yield`.
 
-This type of device power management can be useful when the application is not
-power aware and does not implement runtime device power management. Though,
-:ref:`Device Runtime Power Management <pm-device-runtime-pm>` is the **preferred**
-option for device power management.
+This method of device power management can be useful in the following scenarios:
+
+ - Systems with no device requiring any blocking operations when suspending and
+   resuming. This implementation is reasonably simpler than device runtime
+   power management.
+ - For devices that can not make any power management decision and have to be
+   always active. For example a firmware using Zephyr that is controlled by an
+   external entity (e.g Host CPU). In this scenario, some devices have to be
+   always active and should be suspended together with the SoC when requested by
+   this external entity.
+
+It is important to emphasize that this method has drawbacks (see above) and
+:ref:`Device Runtime Power Management <pm-device-runtime-pm>` is the
+**preferred** method for implementing device power management.
 
 .. note::
```

Comment:
Not really a fan of how having 3 consecutive "notes"  looks like. Maybe you want to create a dedicated "Important considerations" sub-heading and list them 3 as bullet points or something like that?

Reply:
Yep, that is another good suggestion. Can I address it in further pr adding a whole section for this method ?